pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -332,7 +332,7 @@ https://github.com/Darazaki/indent-o-matic/,,
 https://github.com/Yggdroot/indentLine/,,
 https://github.com/ciaranm/inkpot/,,
 https://github.com/jbyuki/instant.nvim/,HEAD,
-https://github.com/pta2002/intellitab.nvim,,
+https://github.com/pta2002/intellitab.nvim/,,
```

Comment:
this commit should ideally be distributed into the previous commits - the update script has an `--add` flag to automatically add plugins

Reply:
Ok I understand now.
I just rebased of master and fixed the `sg-nvim` issue.
The commits should now be properly organized.
Please note, that I had to manually "downgrade" the version of `nvim-tree-lua` by one commit because it was not building properly (duplicated tag error).

Reply:
I made a PR to fix this upstream
https://github.com/nvim-tree/nvim-tree.lua/pull/2066